src/components/i-am-new/NewHere.tsx

```diff
@@ -0,0 +1,12 @@
+import Section from '@/components/Section'
+
+const NewHere = () => {
+  const img="https://res.cloudinary.com/c99/image/upload/v1705049523/Placeholders/Screenshot_2024-01-12_at_10.51.26.png"
+  return (
+    <Section title={'New Here?'} imgSrc={img}>
```

Comment:
feel like the image needs to be smaller (shorter) on mobile, it's taking up too much real estate...